docs/k8s/index.mdx

```diff
@@ -29,77 +11,27 @@ applications run in Kubernetes environments. It supports both [Ingress CRDs](htt
-		<StaticDomainKubernetesGatewayExample />
-	</TabItem>
-</Tabs>
+## When should I use the Kubernetes Operator?
 
-## Route Traffic based on a path
+The Kubernetes Operator is a great tool to use when you want to use ngrok to connect to services running on a Kubernetes Cluster. If you are running Kubernetes and do not want to use the ngrok SDKs, we recommend using the Operator
 
-This would create an Edge on with the Domain `example.ngrok.app` that routes traffic from example.ngrok.app/foo to the Kubernetes service `example-service` running on port 80.
+## Should I use the Ingress Controller or the Gateway API?
 
-<Tabs groupId="k8s" queryString="k8s-basic-path">
-	<TabItem value="ingress" label="Ingress Controller" default>
-		<StaticDomainKubernetesPathExample />
-	</TabItem>
-	<TabItem value="gatewayAPI" label="Gateway API">
-		<StaticDomainKubernetesGatewayPathExample />
-	</TabItem>
-</Tabs>
+The Operator supports both Kubernetes Ingress and the Gateway API. These two different solutions have differences and similarities.
 
-## Modify Headers
+### What is the same?
 
-<Tabs groupId="k8s" queryString="k8s-request-headers">
-	<TabItem value="ingress" label="Ingress Controller" default>
-		<K8sIngressModifyHeadersExample />
-	</TabItem>
-	<TabItem value="gatewayAPI" label="Gateway API">
-		<K8sGatewayModifyHeadersExample />
-	</TabItem>
-</Tabs>
+- Both can route traffic to Kubernetes services
+- Both can take advantage of ngrok's capabilities, including policies, global points of presence, and resiliency
 
-## Redirect Traffic
+### What is different?
 
-<Tabs groupId="k8s" queryString="k8s-request-redirect">
-	<TabItem value="ingress" label="Ingress Controller" default>
-		<K8sIngressRedirectExample />
-	</TabItem>
-	<TabItem value="gatewayAPI" label="Gateway API">
-		<K8sGatewayRedirectExample />
-	</TabItem>
-</Tabs>
+| k8s Ingress                                                            | Gateway API                                                                                                                                                                                                     |
+| ---------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Older, more stable                                                     | Newer, but more experimental                                                                                                                                                                                    |
+| Simple, easier to set up                                               | More complex to set up                                                                                                                                                                                          |
```

Comment:
why is gateway api more complex to set up? this whole section feels strange, like we are trying to convince people to use the ingress controller. IMO, the only reason someone would prefer the ingress controller is if they don't have the gateway apis available in their cluster and don't want to install them. 

also, i think when we talk able similarities and differences, we should talk about how ngrok behaves differently, not the kubernetes experience. things like if edges and policy are configured differently or not available.

Reply:
Gateway API is more complex because there are more things that have to be set up and managed (Gateways configured with the appropriate listeners, Routes for each application), ingress has fewer moving pieces (create ingress and you are done). There are more moving pieces and the technology is newer, which means finding support and solutions is harder. From what I have seen, lots of people are familiar with Ingress since it has been around for a while but a lot of people are not as familiar with Gateway API and don't know which one to use when. This is intended to provide some basics about what the differences between the two technologies are.

There are differences between what you can do with ngrok for the ingress controller and Gateway API, but those differences are temporary while we build the capabilities to make them equivalent, such as adding policy for Gateway API and adding the TCP/TLS Routes. I can call out those differences but eventually the real difference will be in the Kubernetes technologies, not the ngrok capabilities.

Reply:
I removed this section since it seems to be causing confusion.